CHANGELOG.next.asciidoc

```diff
@@ -820,6 +820,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add `fail_on_template_error` option for httpjson input. {pull}24784[24784]
 - Change `okta.target` to `flattened` field type.  {issue}24354[24354] {pull}24636[24636]
 - Added `http.request.id` to `nginx/ingress_controller` and `elasticsearch/audit`. {pull}24994[24994]
+- Add `uri_parts` processor to multiple modules ingest pipelines. {issue}19088[19088] {pull}24699[24699]
```

Comment:
can you add one additional changelog entry that specifies the url encoding change (due to using the `uri_parts` processor) under the "Breaking Changes" section for "Filebeat"? It can say something to the effect of:

> All `url.*` fields apart from `url.original`  in the ___, ____, and ____ modules are now url unescaped due to using the Elasticsearch  `uri_parts` processor. {pull}24699[24699]

After that I'm good with merging this.

Reply:
Will do.

Reply:
I still have the url_decode for the `http.request.referer`. Any thoughts on if I should remove it or keep it?

Reply:
@andrewstucki I will add the change to the log file today or tomorrow. Any thoughts on the question in the above comment??

Reply:
Updated the changelog, just want to finalize the `http.request.referer` question and then we should be good to merge.

Reply:
Sorry for the delayed response, took a look at the ECS docs and they don't appear to specify one way or the other about url decoding. Personally I'd feel like it'd be better to leave them encoded in whatever format they came in, but I could go either way.

Reply:
ok, i'll remove it for now so we can get this merged and can always revisit.

Reply:
@andrewstucki I removed the `url_decode` for `http.request.referrer` field and added the additional entry to the change log.  Should be ready for review.